app/fetcher/hospital_capacities_fetcher.py

```diff
@@ -0,0 +1,78 @@
+import pandas as pd
+
+from app import db, app
+from app.fetcher.fetcher import Fetcher
+from app.models import OckovaniSpotreba, OckovaciMisto, KapacityNemocnic
+
+
+class HospitalCapacitiesFetcher(Fetcher):
+    """
+    Class for updating capacities in hospitals.
+    """
+
+    USED_CSV = 'https://dip.mzcr.cz/api/v1/kapacity-intenzivni-pece-zdravotnicke-zarizeni-10-2021.csv'
+
+    def __init__(self):
+        super().__init__(KapacityNemocnic.__tablename__, self.USED_CSV)
```

Comment:
it was updated after the midnight: `last-modified: Thu, 11 Nov 2021 00:00:02 GMT`

I'm not sure if we want to check modified date. If it's updated before the midnight, fetch will fail.